src/components/Progress/ProgressBar/style/ProgressBar.style.ts

```diff
@@ -0,0 +1,129 @@
+import styled, { css, keyframes } from 'styled-components';
+import { Text } from '../../../Typography';
+import { Position, Type } from '../../../../../lib';
+
+interface StyledProgressProps {
+    positionText?: Position;
+}
+
+interface StyledBarProps {
+    positionText?: Position;
+}
+
+interface StyledFillProps {
+    type?: Type;
+}
+
+interface StyledFillContainerProps {
+    progress: Readonly<number>;
+}
+
+interface StyledProgressTextProps {
+    positionText?: Position;
+}
+
+export const StyledProgressBar = styled.div<StyledProgressProps>`
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    box-sizing: border-box;
```

Comment:
Pecka.. v Atlanticu je lepší si to nastavovat u každé styled componenty zvlášť :)